CHANGES.rst

```diff
@@ -36,6 +36,14 @@ Changes
 
 - Fix BadStatusLine caused by extra `CRLF` after `POST` data #1792
 
+-
+
+- Deprecate undocumented app.on_loop_available signal #1978
+
+-
+
+-
```

Comment:
It reduces the probability of merge conflict.
pytest project used to follow this practice but now they switched to new approach: a folder for change records.
We need the same maybe.